/**:
  ros__parameters:
    offboard:
      controller:
        input:
          setpoint:
            radians: 0.0
        gains:
          k_p: 0.3350 # Proportional gain
          k_d: 0.1616 # Derivative gain
          k_i: 0.3162 # Integral gain
      px4_ros2_message_mapping:
        input: 
          attitude: false
          angular_rate: false
        output:
          servos: true
          motors: false
